rpc/pb/va/va.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto2";
+
+package va;
+
+import "core/core.proto";
+
+service VA {
+	rpc IsSafeDomain(Domain) returns (Valid) {}
+	rpc PerformValidation(PerformValidationRequest) returns (ValidationRecords) {}
+}
+
+message Domain {
+	optional string domain = 1;
+}
+
+message Valid {
+	optional bool valid = 1;
+}
+
+message PerformValidationRequest {
+	optional string domain = 1;
+	optional VAChallenge challenge = 2;
+	optional AuthzMeta authz = 3;
```

Comment:
Since authz.ID and authz.RegistrationID are used just for logging, can we carry them on the `Context`?